server/models.py

```diff
@@ -786,15 +786,18 @@ class CvrBallot(Base):
     record_id = Column(Integer, nullable=False)
     # ballot_position is the counting index of the ballot among all ballots
     # from the batch in the CVR - we use this to match with sampled ballots from the manifest
-    ballot_position = Column(Integer)
+    ballot_position = Column(Integer, nullable=False)
```

Comment:
accidentally left this out from the previous PR

Reply:
grumble, sorry I missed this.